server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -685,6 +685,14 @@ public boolean isES87TSDBCodecEnabled() {
         Property.Final
     );
 
+    public static final FeatureFlag DOC_VALUES_SKIPPER = new FeatureFlag("doc_values_skipper");
```

Comment:
This change will not be here once we merge https://github.com/elastic/elasticsearch/pull/121751